libs/semanticsearch/examples/models/schema/weather_station.schema

```diff
@@ -1,31 +1,58 @@
-model IntPair {
-  key1: IntegerRange(8, 20,4),
-  key2: Int
+specification ai.fetch.weatherstation version 2.1;
+
+model Position { 
+  latitude: Number(-90, 90),
+  longitude: Number(0, 180)  
 };
 
-model Advanced {
-  arr: IntPair,
-  i: Int
+model WeatherStation
+{
+  position: Position,
+  temperature: Number(-180, 300)
 };
 
-// Let's make an instance
-var y : IntPair = {
-    key1: 9,
-    key2: 4
+
+using ai.fetch.weatherstation version 2.1;
+
+instance x : WeatherStation = {
+  position: {
+    latitude: -90,
+    longitude: 0
+  },
+  temperature: -180
```

Comment:
NB: even before the global warming, the lowest temperature recorded in Antarctica appeared only -89 (-93 at ground level, and not exactly on the pole).